template.env

```diff
@@ -71,6 +71,9 @@ NB_RETURN_AGG=true
 NB_API_QUERY_URL=http://XX.XX.XX.XX
 
 # Additional configurable parameters - uncomment to change the defaults
+# Change NB_QUERY_URL_PATH if you're using a custom configuration where the query tool is accessible
+# via a path other than the root (`/`)
+# NB_QUERY_URL_PATH
```

Comment:
🍒 , for clarity:
```suggestion
# Change NB_QUERY_URL_PATH if you're using a proxy server (NGINX, Caddy, etc.) and have configured it so that the query tool is accessible at a path other than the root (`/`). 
# e.g., if your proxy server is set up so that mynode.org/querytool points to the query tool, you would set NB_QUERY_URL_PATH=/querytool
# NB_QUERY_URL_PATH=/
```